index.html

```diff
@@ -0,0 +1,187 @@
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css" integrity="sha512-1ycn6IcaQQ40/MKBW2W4Rhis/DbILU74C1vSrLJxCq57o941Ym01SwNsOMqvEBFlcgUa6xLiPY/NS5R+E6ztJQ==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <link rel="stylesheet" href="style.css">
+  <title>Alex's Concert Page</title>
+</head>
+<body id="home">
+  <header class="header">
+    <nav class="nav1">
+      <ul>
+          <li class="microo">
+            <a rel="noopener" target="_blank" href="https://web.facebook.com/microverseinc/?_rdc=1&_rdr"><i class="fab fa-facebook-f"></i></a>
+          </li>
+          <li class="microo">
+            <a rel="noopener" target="_blank" href="https://twitter.com/microverseinc"><i class="fab fa-twitter"></i></a>
+          </li>
+          <li>English</li>
+          <li>My Page</li>
+          <li>Logout</li>
+      </ul>
+  </nav>
+    <nav class="head-all">
+      <i class="fas fa-bars menu"></i>
+      <ul>
+        <li class="logo-img">
+          <a href="./index.html">
+            <img src="./images/logo-music.png" alt="">
+          </a>
+        </li>
+        <li>
+          <a href="./index-about.html"><h2>About</h2></a>
+        </li>
+        <li>
+          <a href="#program"><h2>Program</h2></a>
+        </li>
+        <li>
+          <a href="#guests"><h2>Guests</h2></a>
+        </li>
+        <li>
+          <a href="#partners"><h2>Partners</h2></a>
+        </li>
+        <li class="campagn">
+          <a href="#"><h2>MC Campaign</h2></a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+  <main>
+    <section class="menu-sec">
+      <nav>
+        <a href="./index.html"><h2 class="links">Home</h2></a>
+        <a href="./index-about.html"><h2 class="links">About</h2></a>
+        <a href="#program"><h2 class="links">Program</h2></a>
+        <a href="#guests"><h2 class="links">Guests</h2></a>
+        <a href="#partners"><h2 class="links">Partners</h2></a>
+      </nav>
+      <div class="cancel links">
+        &times;
+      </div>
+    </section>
+    <article>
+      <section class="sec1">
+        <p class="para1">
+          "Come One Come All"
+        </p>
+        <div class="title">
+          <h1 class="main-title">
+            <span>CULTURAL</span> MUSIC AFRICA CONCERT 2021
+          </h1>
+        </div>
+        <P class="para2">
+          Music is an essential element of culture. Sharing music from one culture to another gives people an insight into another way of life. All are invited from all countries in taking places in June, in Kenya
+        </P>
+        <h2 class=" year">
+          2021.06.20(FRI) ~ 21(SAT)
+        </h2>
+        <p class="para2 differ">
+          @Nyayo Stadium in Kenya, Cultural center and more
+        </p>
```

Comment:
- [ ] Please adjust the font color and size to look familiar to that of the original design.
### Your Implementation
![image](https://user-images.githubusercontent.com/45460192/192044195-d5e5cec9-be78-4257-aba6-5f11f0c7613c.png)
### Required
![image](https://user-images.githubusercontent.com/45460192/192044119-207cc16c-ce7e-498e-8962-610b8d0fdced.png)